lib/api/fields/assets/js/media.js

```diff
@@ -172,15 +172,15 @@
 
 			});
 
-			this.container.on( 'click', '.dashicons-post-trash', function( e ) {
+			this.container.on( 'click', '.bs-button-trash', function( e ) {
 
 				e.preventDefault();
 
 				$this.deleteImage( $(this) );
 
 			});
 
-			this.container.on( 'click', '.dashicons-edit', function( e ) {
+			this.container.on( 'click', '.bs-button-edit', function( e ) {
```

Comment:
General suggestion - use namespaces for events, so one can easily remove _just_ Beans events: